extra/man/alacritty.5.scd

```diff
@@ -0,0 +1,800 @@
+	Default: _true_
+
+*class* { instance = <string>, general = <string> }
+
+	Window class _(Linux/BSD only)_
+
+	Default: _{ instance = "Alacritty", general = "Alacritty" }_
+
+*decoration_theme_variant* "Dark" | "Light" | "None"
+
+	Override the variant of the System theme/GTK theme/Wayland client side
+	decorations. Set this to _"none"_ to use the system's default theme variant.
+
+	Default: _"None"_
+
+*resize_increments* <boolean>
+
+	Prefer resizing window by discrete steps equal to cell dimensions.
+
+	Default: _false_
+
+*option_as_alt* "OnlyLeft" | "OnlyRight" | "Both" | "None"
+
+	Make _Option_ key behave as _Alt_ _(macOS only)_
+
+	Default: _"None"_
+
+# Scrolling
+
+This section documents the *[scrolling]* table of the configuration file.
+
+*history* <integer>
+
+	Maximum number of lines in the scrollback buffer.++
+Specifying _0_ will disable scrolling.++
+Limited to _100000_.
+
+	Default: _10000_
+
+*multiplier* <integer>
+
+	Number of line scrolled for every input scroll increment.
+
+	Default: _3_
+
+# Font
+
+This section documents the *[font]* table of the configuration file.
+
+*normal* { family = <string>, style = <string> }
+
+	Default:
+		Linux/BSD: _{ family = "monospace", style = "Regular" }_++
+Windows:   _{ family = "Consolas",  style = "Regular" }_++
+macOS:     _{ family = "Menlo",     style = "Regular" }_
+
+*bold* { family = <string>, style = <string> }
+
+	If the family is not specified, it will fall back to the value specified for
+	the normal font.
+
+	Default: _{ style = "Bold" }_
+
+*italic* { family = <string>, style = <string> }
+
+	If the family is not specified, it will fall back to the value specified for
+	the normal font.
+
+	Default: _{ style = "Italic" }_
+
+*bold_italic* { family = <string>, style = <string> }
+
+	If the family is not specified, it will fall back to the value specified for
+	the normal font.
+
+	Default: _{ style = "Bold Italic" }_
+
+*size* <float>
+
+	Default: _11.0_
+
+*offset* { x = <integer>, y = <integer> }
+
+	Offset is the extra space around each character. _y_ can be thought of as
+	modifying the line spacing, and _x_ as modifying the letter spacing.
+
+	Default: _{ x = 0, y = 0 }_
+
+*glyph_offset* { x = <integer>, y = <integer> }
+
+	Glyph offset determines the locations of the glyphs within their cells with
+	the default being at the bottom. Increasing _x_ moves the glyph to the
+	right, increasing _y_ moves the glyph upward.
+
+*builtin_box_drawing* <boolean>
+
+	When _true_, Alacritty will use a custom built-in font for box drawing
+	characters (Unicode points 2500 - 259f).
+
+	Default: _true_
+
+# Colors
+
+This section documents the *[colors]* table of the configuration file.
+
+Colors are specified using their hexadecimal values with a _#_ prefix: _#RRGGBB_
+
+*primary*
+
+	*foreground* <string>
+
+		Default: _"#1d1f21"_
+
+	*background* <string>
+
+		Default: _"#c5c8c6"_
+
+	*dim_foreground* <string>
+
+		If the bright foreground color is not set, it is automatically
```

Comment:
```suggestion
		If the dim foreground color is not set, it is automatically
```

I think it's `dim`?

Reply:
Also, the default in the config is `None`, when you look into color.rs, so it could be changed either here or in the configuration to specify the value explicitly. It's also a bit weird that we have `Dim` colors defined in the config, but not use `None`.

it's also weird that we have `Dim` colors defined in the `color.rs`, but they actually never used, but still being computed on demand.